cms/templates/js/course-outline.underscore

```diff
@@ -57,11 +91,42 @@ if (staffOnlyMessage) {
         addStatusMessage(messageType, messageText);
     }
 
-    if (hasPartitionGroups) {
+    if (hasPartitionGroups && numAllowedGroups === 0 && xblockInfo.isVertical()) {
         addStatusMessage(
             'partition-groups',
             gettext('Access to some content in this unit is restricted to specific groups of learners')
         );
+    } else if (numAllowedGroups >= 1 && xblockInfo.isVertical()) {
+        var selectedGroupsLabel = '';
+
+        for (var schemeId in groupAccessMap) {
```

Comment:
I'm a bit confused here-- you are looping over all schemes? The selected groups should all be within a single scheme.